runner/Dockerfile

```diff
@@ -110,7 +110,8 @@ RUN mkdir /opt/hostedtoolcache \
     && chgrp docker /opt/hostedtoolcache \
     && chmod g+rwx /opt/hostedtoolcache
 
-COPY entrypoint.sh /
+COPY entrypoint.sh job_start.sh job_end.sh update_status.sh /
+RUN chmod 755 entrypoint.sh job_start.sh job_end.sh update_status.sh
```

Comment:
Why did you need to chmod entrypoint.sh too? (I thought it's already executable by any user/group

Reply:
removed - wasn't needed 👍